src/ui/public/vis/map/service_settings.js

```diff
@@ -20,11 +20,13 @@
 import { uiModules } from '../../modules';
 import _ from 'lodash';
 import MarkdownIt from 'markdown-it';
+import { defaultLayerLoadWarning } from './map_messages';
 import { ORIGIN } from '../../../../legacy/core_plugins/tile_map/common/origin';
-import { EMSClientV66 } from '../../../../legacy/core_plugins/tile_map/common/ems_client';
+import { EMSClientV66 } from
+  '../../../../legacy/core_plugins/tile_map/common/ems_client';
+
 
 const markdownIt = new MarkdownIt({
-  html: false,
```

Comment:
I think we should keep this. Every other instantiation of MarkdownIt specifies html as false. For security reasons, it has to be false. Specifying false does not hurt anything and provides the following benefits
1) ensures it will be false if the default value ever changes.
2) lets the reader know html is false without having to look-up documenation for markdownIt and track down default configurations.